Marlin/src/module/temperature.cpp

```diff
@@ -54,32 +86,62 @@
   #endif
   #ifndef MAX31865_MOSI_PIN
     #define MAX31865_MOSI_PIN MOSI_PIN
-  #endif
-  #ifndef MAX31865_MISO_PIN
+  #endif  
+  #if !defined(MAX31865_MISO_PIN) && PIN_EXISTS(MAX6675_DO)
```

Comment:
I think `PIN_EXISTS(MAX6675_DO)` already validade if macro is defined, you can simply it here.

Reply:
Yes, PIN_EXISTS will check to see if MAX6675_DO_PIN is defined and has a pin assigned to it.  But PIN_EXISTS in that statement does not check to see if MAX31865_MISO_PIN is defined.... MAX6675_DO_PIN is not equal to MAX31865_MISO_PIN when that statement is hit.